pkgs/applications/graphics/processing/default.nix

```diff
@@ -1,73 +1,78 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, fetchurl, xmlstarlet, makeWrapper, ant, jdk, rsync, javaPackages, libXxf86vm, gsettings-desktop-schemas }:
+{ lib, stdenv, fetchFromGitHub, fetchurl, ant, unzip, makeWrapper, jdk, javaPackages, rsync, ffmpeg, batik, gsettings-desktop-schemas, xorg, wrapGAppsHook }:
+let
+  vaqua = fetchurl {
+    name = "VAqua9.jar";
+    url = "https://violetlib.org/release/vaqua/9/VAqua9.jar";
+    sha256 = "cd0b82df8e7434c902ec873364bf3e9a3e6bef8b59cbf42433130d71bf1a779c";
+  };
+
+  jna = fetchurl {
+    name = "jna-5.10.0.zip";
+    url = "https://github.com/java-native-access/jna/archive/5.10.0.zip";
+    sha256 = "B5CakOQ8225xNsk2TMV8CbK3RcsLlb+pHzjaY5JNwg0=";
+  };
+
+  flatlaf = fetchurl {
+    name = "flatlaf-2.4.jar";
+    url = "https://repo1.maven.org/maven2/com/formdev/flatlaf/2.4/flatlaf-2.4.jar";
+    sha256 = "NVMYiCd+koNCJ6X3EiRx1Aj+T5uAMSJ9juMmB5Os+zc=";
+  };
 
+in
 stdenv.mkDerivation rec {
   pname = "processing";
-  version = "3.5.4";
+  version = "1286-4.0.1";
```

Comment:
Isn't 1286 just the build number?

Reply:
Yes, I've changed it to just be 

```
version = "4.1.1";
```
and have a separate variable for the `buildNumber`.